src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-color: black;
+}
+
+#qa-block {
+  text-align: left;
+}
+
+.first-square {
+  width: 328px;
+  margin: 20px 20px 40px;
+  padding: 10px;
+  border: 1px solid #e75551;
+  background-color: #308dc0;
+}
+
+.second-square {
+  border: 1px solid #e75551;
+  background-color: #ffa500;
+  padding: 10px;
+}
+
+.third-square {
+  border: 1px solid #e75551;
+  background-color: #ddd;
+  padding: 10px;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.